src/bundle/Resources/public/js/scripts/sidebar/extra.actions.js

```diff
@@ -20,13 +20,18 @@
                 const focusElement = actions.querySelector(btn.dataset.focusElement);
                 const detectClickOutside = (event) => {
                     const isNotButton = event.target !== btn || !btn.contains(event.target);
+                    const isNotButtons = !btns.includes(event.target);
```

Comment:
Maybe `shouldCollapseMenu`? Because we have `isNotButton` and `isNotButtons`, this is misleading.